src/main/java/weightScheme/OrderedWeight.java

```diff
@@ -71,4 +71,9 @@ public double getTotalWeight() {
                 .mapToDouble(weight -> weight.getTotalWeight())
                 .sum();
     }
+
+    @Override
+    public double getWeightOfEachInstance() {
+        return orderedWeights[0].getWeightOfEachInstance();
```

Comment:
This function does not work and only returns the weight of the highest impact work.